plugins/inputs/nats_top/nats_top.go

```diff
@@ -0,0 +1,93 @@
+package natstop
+
+import (
+	"fmt"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"time"
+
+	"encoding/json"
+
+	"github.com/influxdata/telegraf"
+	"github.com/influxdata/telegraf/plugins/inputs"
+
+	gnatsd "github.com/nats-io/gnatsd/server"
+)
+
+type NatsTop struct {
+	Server string
+}
+
+var sampleConfig = `
+  ## The address of the monitoring end-point of the NATS server
+  server = "http://localhost:1337"
```

Comment:
TLS support could be nice, with the normal HTTP options such as in the apache input.

Reply:
That's a great idea, will add.